akka-actor-tests/src/test/scala/akka/event/LoggingReceiveSpec.scala

```diff
@@ -118,6 +118,23 @@ class LoggingReceiveSpec extends WordSpec with BeforeAndAfterAll {
       }
     }
 
+    "log with MDC" in {
+      new TestKit(appLogging) {
+        system.eventStream.subscribe(testActor, classOf[Logging.Debug])
+        val myMDC = Map("hello" → "mdc")
+        val a = system.actorOf(Props(new Actor with DiagnosticActorLogging {
+          override def mdc(currentMessage: Any) = myMDC
+          def receive = LoggingReceive {
+            case "hello" ⇒
+          }
+        }))
+        a ! "hello"
+        expectMsgPF(1 second, "Logging.Debug2") {
```

Comment:
too short timeout, see https://github.com/akka/akka/blob/master/CONTRIBUTING.md#preferred-ways-to-use-timeouts-in-tests